.github/workflows/sync-issue-labels.yml

```
@@ -28,8 +30,8 @@ jobs:
             
               // Fetch labels from the linked issue
               const issueResponse = await github.rest.issues.get({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
+                owner,
+                repo,
                 issue_number: issueNumber,
               });
             
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)